src/haz3lweb/Store.re

```diff
@@ -18,6 +20,8 @@ let key_to_string =
   | Mode => "MODE"
   | Scratch => "SAVE_SCRATCH"
   | Documentation => "SAVE_DOCUMENTATION"
+  | Tutorial(id) => Haz3lcore.Id.to_string(id)
```

Comment:
prefix with "TUTORIAL" so it is clear when debugging what this entry is, not just a random ID